.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and run tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 12 ]
+
+    steps:
+    - name: Clone repository
+      uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Cache node modules
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-v${{ matrix.node }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}-${{ hashFiles('.github/workflows/test.yml') }}
```

Comment:
Is it necessary to add hashes of both `package.json` and `package-lock.json`?

Reply:
(Copied from Bootstrap.) I guess that the hash of `package-lock.json` should be enough as `npm ci` installs from lockfile, not looking at `package.json` at all.